### DISCUSSION TITLE
HF patch for AI Shoujo takes over 3 days to download

### DISCUSSION THREAD

**Discussion by DeltaAgent26** [community/community#42211] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hello @ManlyMarco,

I have gigabit internet both up and down... However, the patch only updates at the most 350KB/s
This last update was/is over 65 GB, and I have been DL-ing the update for nearly 3 days, and still have over a day left on the progress bar.

Is the server for these particular updates throttled as far as download speeds?
And is there a way to increase the speed at which the updates can be downloaded?

Thanks.

**Comment by ManlyMarco** [comment, marked as answer]
I assume you mean the mod updater and not the patch itself? The servers are perpetually overloaded because people keep downloading everything even if they don't use studio.
You don't actually need to run that update once you have the patch, in practice it most likely won't give you anything new unless you need the new mods for a card or scene you downloaded or you need specific items for your studio scene or characters. Also you might encounter new bugs that were not found when testing the patch.

**Reply by DeltaAgent26** [reply]
Yes the mod updater. The mods themselves. As I said it is extremely slow to download the updates.
The download speeds are abysmal. Considering the last update, as I said was over 65 Gigabytes,,,
And the fastest speed I saw during the download was about 350KB/s... so it took over 8 days....

But yes, I will ignore the update notifications from now on, unless it is something I specifically need. 
Thank you.
